packages/unmock-core/src/generator.ts

```diff
@@ -302,7 +302,7 @@ const generateMockFromTemplate = (
   jsf.option("useDefaultValue", false);
   const resolvedTemplate = jsf.generate(template);
 
-  const body = JSON.stringify(resolvedTemplate);
+  const body = resolvedTemplate;
```

Comment:
I think `const body = jsf.generate(template);` would be better (+ removing the `resolvedTemplate` above).
Or then returning `{ body: resolvedTemplate }`, etc, but this assignment is no longer needed :)